src/Symfony/Bundle/SecurityBundle/Resources/config/security.php

```diff
@@ -0,0 +1,275 @@
+                abstract_arg('LogoutListener'),
+                abstract_arg('FirewallConfig'),
+            ])
+
+        ->set('security.firewall.lazy_context', LazyFirewallContext::class)
+            ->abstract()
+            ->args([
+                [],
+                service('security.exception_listener'),
+                abstract_arg('LogoutListener'),
+                abstract_arg('FirewallConfig'),
+                service('security.untracked_token_storage'),
+            ])
+
+        ->set('security.firewall.config', FirewallConfig::class)
+            ->abstract()
+            ->args([
+                abstract_arg('name'),
+                abstract_arg('user_checker'),
+                abstract_arg('request_matcher'),
+                false, // security enabled
+                false, // stateless
+                null,
+                null,
+                null,
+                null,
+                null,
+                [], // listeners
+                null, // switch_user
+            ])
+
+        ->set('security.logout_url_generator', LogoutUrlGenerator::class)
+            ->args([
+                service('request_stack')->nullOnInvalid(),
+                service('router')->nullOnInvalid(),
+                service('security.token_storage')->nullOnInvalid(),
+            ])
+
+        // Provisioning
+        ->set('security.user.provider.missing', MissingUserProvider::class)
+            ->abstract()
+            ->args([
+                abstract_arg('firewall'),
+            ])
+
+        ->set('security.user.provider.in_memory', InMemoryUserProvider::class)
+            ->abstract()
+
+        ->set('security.user.provider.ldap', LdapUserProvider::class)
+            ->abstract()
+            ->args([
+                abstract_arg('security.ldap.ldap'),
+                abstract_arg('base dn'),
+                abstract_arg('search dn'),
+                abstract_arg('search password'),
+                abstract_arg('default_roles'),
+                abstract_arg('uid key'),
+                abstract_arg('filter'),
+                abstract_arg('password_attribute'),
+                [], // extra_fields (email etc)'),
+            ])
+
+        ->set('security.user.provider.chain', ChainUserProvider::class)
+            ->abstract()
+
+        ->set('security.http_utils', HttpUtils::class)
+            ->args([
+                service('router')->nullOnInvalid(),
+                service('router')->nullOnInvalid(),
+            ])
+        ->alias(HttpUtils::class, 'security.http_utils')
+
+        // Validator
+        ->set('security.validator.user_password', UserPasswordValidator::class)
+            ->tag('validator.constraint_validator', ['alias' => 'security.validator.user_password'])
+            ->args([
+                service('security.token_storage'),
+                service('security.encoder_factory'),
+            ])
+
+        // Cache
+        ->set('cache.security_expression_language')
```

Comment:
public=false is missing. outwise it will inherit the public=true from the parent